src/merkle/partial_mt/mod.rs

```diff
@@ -0,0 +1,306 @@
+use super::{
+    BTreeMap, BTreeSet, MerkleError, MerklePath, NodeIndex, Rpo256, RpoDigest, ValuePath, Vec,
+    Word, EMPTY_WORD,
+};
+
+extern crate alloc;
+use alloc::format;
```

Comment:
I'd use conditional imports similar to how it is done [here](https://github.com/facebook/winterfell/blob/main/utils/core/src/collections.rs#L12). Also, I'd move this to the utils module so that we could use it in other places as well if we need to.